### PR TITLE
feat(analytics): use a different tracking id in production

### PR DESCRIPTION
**Pull request by oliviertassinari**
- `UA-89989315-1`: production
- `UA-89989315-2`: development

### CODE REVIEW

**Comment by neoziro** [comment]
Ca sert à quelque chose d'envoyer les données en dev?

**Comment by neoziro** [comment]
@oliviertassinari is it useful to send data in development?

File: package.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "server:development": "nodemon src/server/bootstrap/bootstrap",
+    "server:development": "NODE_ENV=development nodemon src/server/bootstrap/bootstrap",
```
Comment (neoziro) [inline package.json]: Useless, default env will be development, we must get environment only from `config.get('env')`.

**Comment by oliviertassinari** [comment]
The idea is to be closer to the production with a full end-to-end approach.
Not sending data mean we can't test stuff on the Google Analytics interface.

**Pull request merged**